docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,479 @@
+spec:
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routes:
+  - destinationCIDRBlock: 0.0.0.0/0
+    natGatewayRef:
+      from:
+        name: tutorial-natgateway1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: tutorial-public-subnet1
+spec:
+  availabilityZone: us-west-2a
+  cidrBlock: 10.0.0.0/20
+  mapPublicIPOnLaunch: true
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routeTableRefs:
+  - from:
+      name: tutorial-public-route-table
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: tutorial-private-subnet1
+spec:
+  availabilityZone: us-west-2a
+  cidrBlock: 10.0.128.0/20
+  vpcRef:
+    from:
+      name: tutorial-vpc
+  routeTableRefs:
+  - from:
+      name: tutorial-private-route-table-az1
+---
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: SecurityGroup
+metadata:
+  name: tutorial-security-group
+spec:
+  description: "ack security group"
+  name: tutorial-sg
+  vpcRef:
+     from:
+       name: tutorial-vpc
+  ingressRules:
+    - ipProtocol: tcp
+      fromPort: 22
+      toPort: 22
+      ipRanges:
+        - cidrIP: "0.0.0.0/0"
+          description: "ingress"
+```
+ 
+The output should look similar to:
+```
+vpc.ec2.services.k8s.aws/tutorial-vpc created
+internetgateway.ec2.services.k8s.aws/tutorial-igw created
+natgateway.ec2.services.k8s.aws/tutorial-natgateway1 created
+elasticipaddress.ec2.services.k8s.aws/tutorial-eip1 created
+routetable.ec2.services.k8s.aws/tutorial-public-route-table created
+routetable.ec2.services.k8s.aws/tutorial-private-route-table-az1 created
+subnet.ec2.services.k8s.aws/tutorial-public-subnet1 created
+subnet.ec2.services.k8s.aws/tutorial-private-subnet1 created
+securitygroup.ec2.services.k8s.aws/tutorial-security-group created
+```
+
+* Check the CRD's installed using `kubectl get crds`:
+
+The output should look similar to:
+```
+NAME                                         CREATED AT
+adoptedresources.services.k8s.aws            2022-10-15T01:58:26Z
+dhcpoptions.ec2.services.k8s.aws             2022-10-15T01:58:26Z
+elasticipaddresses.ec2.services.k8s.aws      2022-10-15T01:58:26Z
+eniconfigs.crd.k8s.amazonaws.com             2022-09-30T23:00:32Z
+fieldexports.services.k8s.aws                2022-10-15T01:58:26Z
+instances.ec2.services.k8s.aws               2022-10-15T01:58:27Z
+internetgateways.ec2.services.k8s.aws        2022-10-15T01:58:27Z
+natgateways.ec2.services.k8s.aws             2022-10-15T01:58:27Z
+routetables.ec2.services.k8s.aws             2022-10-15T01:58:27Z
+securitygrouppolicies.vpcresources.k8s.aws   2022-09-30T23:00:35Z
+securitygroups.ec2.services.k8s.aws          2022-10-15T01:58:28Z
+subnets.ec2.services.k8s.aws                 2022-10-15T01:58:28Z
+transitgateways.ec2.services.k8s.aws         2022-10-15T01:58:28Z
+vpcendpoints.ec2.services.k8s.aws            2022-10-15T01:58:28Z
+vpcs.ec2.services.k8s.aws                    2022-10-15T01:58:28Z
+```
```

Comment:
You might want to move this into the section above `Install ACK EC2-Controller`, since it's not really relevant to the single YAML manifest example you are doing here.